GameData/RealismOverhaul/RO_Comms.cfg

```diff
@@ -33,3 +33,13 @@
 {
 	!useAntennaConfig = DEL
 }
+
+//  ==================================================
+//  Remove All RemoteTech Related Configs.
+//  ==================================================
+
+@PART:[*]:FOR[zzzRealismOverhaul]
+{
+    !MODULE[ModuleRTAntenna*],{}
+    !MODULE[ModuleSPU*],*{}
+}
```

Comment:
Revert this, please.  While there does not appear much interest in maintaining RT patches in the current RO community, we should not delete support.